app/controllers/account/billing/invoices/list.js

```diff
@@ -0,0 +1,155 @@
+import Controller from '@ember/controller';
+import EmberTableControllerMixin from 'open-event-frontend/mixins/ember-table-controller';
+
+export default class extends Controller.extend(EmberTableControllerMixin) {
+
+upcomingInvoiceColumns = [
+  {
+    name      : 'Invoice ID',
+    valuePath : 'invoiceId'
+  },
+  {
+    name      : 'Event Name',
+    valuePath : 'eventName'
+  },
+  {
+    name      : 'Date Issued',
+    valuePath : 'dateIssued'
+  },
+  {
+    name      : 'Outstanding Amount',
+    valuePath : 'amount'
+  },
+  {
+    name      : 'View Invoice',
+    valuePath : 'invoiceLink'
+  }
+];
+
+paidInvoiceColumns = [
+  {
+    name      : 'Invoice ID',
+    valuePath : 'invoiceId'
+  },
+  {
+    name      : 'Event Name',
+    valuePath : 'eventName'
+  },
+  {
+    name      : 'Date Issued',
+    valuePath : 'dateIssued'
+  },
+  {
+    name      : 'Amount',
+    valuePath : 'amount'
+  },
+  {
+    name      : 'Date Paid',
+    valuePath : 'datePaid'
+  },
+  {
+    name      : 'View Invoice',
+    valuePath : 'invoiceLink'
+  }
+
+];
+
+dueInvoiceColumns = [
+  {
+    name      : 'Invoice ID',
+    valuePath : 'invoiceId'
+  },
+  {
+    name      : 'Event Name',
+    valuePath : 'eventName'
+  },
+  {
+    name      : 'Date Issued',
+    valuePath : 'dateIssued'
+  },
+  {
+    name      : 'Amount Due',
+    valuePath : 'amount'
+  },
+  {
+    name      : 'View Invoice',
+    valuePath : 'invoiceLink'
+  }
+
+];
+allInvoiceColumns =  [
```

Comment:
make them computed properties.

Reply:
@CosmicCoder96 @niranjan94 Had mentioned that computed properties won't be required here. POJOs are enough. [In one of his reviews]